recipe/meta.yaml

```diff
@@ -1,27 +1,28 @@
 {% set name = "prophet" %}
-{% set version = "1.1.4" %}
+{% set version = "1.1.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7f91e4ba322786e9512158783800b6eafb002200a7b3e9eadc13d7ff4d6e0634
-  patches:
-    - setup-make-folder.patch
+  sha256: 80973c0b8a22d835bfa9d6665a78ebc63115135eaef0f73b46ee14e9bad3ca1a
 
 build:
   number: 0
   # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
   # https://blog.quantinsti.com/installing-prophet-library-windows/
   skip: true  # [py<37 or (linux and s390x)]
+  # because of missing numpy
+  skip: true  # [py>311]
```

Comment:
This is not correct. Your host should have:
```
  - numpy 1.22 # [py<311]
  - numpy {{ numpy }} # [py>=311]
```
If you do even need numpy in host.